README.md

```diff
@@ -3,11 +3,12 @@ Automation Tools
 
 The Automation Tools project is a set of python scripts, that are designed to automate the processing of transfers in an Archivematica pipeline.
 
-Currently, the only automation tool is automate transfers.  It is used to prepare transfers, move them into the pipelines processing location, and take actions when user input is required.  Only one transfer is sent to the pipeline at a time, the scripts wait until the current transfer is resolved (failed, rejected or stored as an AIP) before automatically starting the next available transfer. 
+Currently, the only automation tool is automate transfers.  It is used to prepare transfers, move them into the pipelines processing location, and take actions when user input is required.  Only one transfer is sent to the pipeline at a time, the scripts wait until the current transfer is resolved (failed, rejected or stored as an AIP) before automatically starting the next available transfer.
 
 The code is available on [Github](http://github.com/artefactual/automation-tools).
 
 The code is deployed to `/usr/lib/archivematica/automation-tools`.
+Configuration files should go into `/etc/archivematica/automation-tools/`, the automation tools require a configuration file named `automation-tools.conf` in this directory to run (a sample file is provided in the `etc/` directory of this repository).
```

Comment:
Nitpick: I think this makes more sense split up more.  eg "Configuration files should go into `/etc/archivematica/automation-tools/`.  A configuration file named `automation-tools.conf` is required, and a sample is provided..."